compiler/src/dotty/tools/dotc/transform/CheckUnused.scala

```diff
@@ -498,6 +505,23 @@ object CheckUnused:
     end getUnused
     //============================ HELPERS ====================================
 
+
+    /**
+     * Checks if import selects a def that is transparent and inline
+     */
+    private def isTransparentAndInline(imp: tpd.Import)(using Context): Boolean =
+      (for {
```

Comment:
One last nitpick: this `for` is literally a map. So instead of having `.exists(identity)` at the end, you can just do `imp.selectors.exists(/*body of for*/)`